torch/utils/cpp_extension.py

```diff
@@ -1629,8 +1629,15 @@ def load_inline(name,
         ...                      functions=['sin_add'])
 
     .. note::
-        By default, the Ninja backend uses #CPUS + 2 workers to build the
-        extension. This may use up too many resources on some systems. One
+        Since load_inline will just-in-time compile the source code, please ensure
+        that you have the right toolchains installed in the runtime. For example,
+        if you are loading C++, make sure a C++ compiler is available. If you're loading
```

Comment:
> if you are loading C++

Why the "if", isn't this always loading c++ code?

Reply:
okay yea